index.html

```diff
@@ -104,7 +111,37 @@ <h2>-ui</h2>
           </ul>
         </div>
         <div class="main">
+          <div class="social-links">
+            <div class="nav-group">
+              <div class="nav-item">
+                <a target="_blank" href="https://twitter.com/kunal_tijare"
+                  ><i class="fa fa-twitter"></i
+                ></a>
+              </div>
+              <div class="nav-item">
+                <a
+                  target="_blank"
+                  href="https://github.com/Dev-kunal/Components-Library--DashUi"
+                  ><i class="fa fa-github"></i
+                ></a>
+              </div>
+            </div>
+          </div>
           <ul class="component-list">
+            <li id="getting-started">
+              <h2 class="component-name">Getting Started</h2>
+              <article class="component component-list component-center">
+                <strong
+                  >Add this in &lt;head/&gt; of your HTML and youre ready to
+                  Go..🏹🚀</strong
+                >
+                <pre>
+                  <code class="html">
+&lt;link rel=&quot;stylesheet&quot; href=&quot;https://dashedui.netlify.app/Styles/components.css&quot;&gt;
+                  </code>
+                </pre>
+              </article>
+            </li>
             <li id="alert">
               <h2 class="component-name">Alert</h2>
```

Comment:
Close button in alert and dismiss card should ideally be a button element like in modal and toast.

Reply:
Thanks for the review @villdev. i worked on your suggestions. improved the color contrast, add labels for input and button in the card.